onadata/apps/api/viewsets/user_profile_viewset.py

```diff
@@ -303,6 +330,16 @@ def partial_update(self, request, *args, **kwargs):
 
             profile.metadata = metadata
             profile.save()
+
+            # send notification on updating user profile
+            send_message(
```

Comment:
`partial_update` will call the serializer's `save` method. If we refactor  to send the message for `update` within the serializer, we don't need this change